static/js/component.boite-accordeon.js

```diff
@@ -30,6 +30,12 @@ Vue.component('boite-accordeon', {
 							:valeur="formatterNombre(terrain['surface_terrain']) + ' m²'"
 							icone="fa fa-tree"
 							:texte="terrain['nature_culture'] + (terrain['nature_culture_speciale'] != 'None' ? ' / ' + terrain['nature_culture_speciale'] : '')">
+						</boite>
+						<boite <!-- for terrains a batir, display price per square meter -->
+							v-for="terrain in mutation.terrains"
+							:valeur="formatterNombre(mutation.infos[0]['valeur_fonciere'] / terrain['surface_terrain']) + ' €/m²'"
+							icone="fa fa-tree"
+							:texte="terrain['nature_culture'] + (terrain['nature_culture_speciale'] != 'None' && terrain['nature_culture'] != 'sols' && terrain['nature_culture_speciale'] != 'sols' ? ' / ' + terrain['nature_culture_speciale'] : '')">
```

Comment:
The last condition look weird:

	terrain['nature_culture_speciale'] != 'sols' ? ' / ' + terrain['nature_culture_speciale'] : ''